src/components/tools/media/ResourceList.js

```diff
@@ -1,101 +1,45 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import ResourceFeatureItem from '../../common/ResourceFeatureItem';
-import { assetUrl } from '../../../lib/assetUtil';
-
-
-const localMessages = {
-
-  globalCollectionsTitle: { id: 'marketing.globalCollections.title', defaultMessage: 'The South African Digital News Report 2020' },
-  globalCollectionsDescription: { id: 'marketing.globalCollections.description', defaultMessage: '<p>Produced in collaboration with Reuters Institute for the Study of Journalism, Oxford University.<br/>  <a href="https://reutersinstitute.politics.ox.ac.uk/sites/default/files/2020-06/DNR_2020_FINAL.pdf">Download the full DNR 2020 report</a> </p>' },
-
-  saElectionsTitle: { id: 'marketing.globalCollections.title', defaultMessage: 'A race to the bottom' },
-  saElectionsDescription: { id: 'marketing.globalCollections.description', defaultMessage: `<h3>Populism, mis/disinformation and South Africa’s 2021 election</h3><p>An overview and analysis of the roles populism and mis/disinformation played in the run-up to South Africa’s 2021 Local Government Elections.<br/>Produced by the South African Anti-Disinformation Project.<br/> <a href="${assetUrl('/static/docs/A race to the bottom _ SA Elections 2021 _Disinfo Project.pdf')}" target = "_blank">Download the report</a> </p>` },
```

Comment:
This is what I was asking; why not convert what we're getting from the CMS to this kind of format (descriptions are just strings) so that 🤞🏽, we don't need to change any other part of the app?

Reply:
I see, I could revert to this approach, at first I wanted to make an implementation like this so that we have a switch with minimum effort. However, I wanted to re-use the [RichText](https://github.com/CodeForAfrica/web-tools/blob/feat/web-tools-cms/src/components/common/RichText.js) component we already have in the ui repo and it implied that I would still need to make changes within the `ResourceList` to know when to use it (so that I don't break other apps)

Thus, I went ahead and injected whatever is coming from payload to the component while maintaining a flag that will distinguish the type of content we are parsing at a given time.

If we are to maintain the exact structure the only thing I would need is to convert the RichText component to a function that will parse what is coming from Payload and return HTML (could be out of habit but my preference as I was doing the implementation was having the HTML conversion as the last step through the reused component)

Reply:
Given that MC have already migrated to `v2`, and will no longer be using/maintaining this `v1` app, I think our goal should be to change the code _as little as possible_ to improve quality of life for our users (🤞🏽 within the next new months, CivicSignal will figure out the way forward; whether to migrate to `v2` or using `v1` for the foreseeable future).

This is why I believe keeping the structure as is is probably the best solution for now.

Reply:
Sure let me quickly make necessary adjustments, I'll maintain the basic structure for the page 👍